src/RestrictedPython/Guards.py

```diff
@@ -191,35 +182,43 @@
 
 def _write_wrapper():
     # Construct the write wrapper class
+
     def _handler(secattr, error_msg):
         # Make a class method.
+
         def handler(self, *args):
             try:
                 f = getattr(self.ob, secattr)
             except AttributeError:
                 raise TypeError(error_msg)
+
             f(*args)
+
         return handler
 
     class Wrapper(object):
+
         def __init__(self, ob):
             self.__dict__['ob'] = ob
 
         __setitem__ = _handler(
             '__guarded_setitem__',
-            'object does not support item or slice assignment')
+            'object does not support item or slice assignment',
+        )
 
         __delitem__ = _handler(
             '__guarded_delitem__',
-            'object does not support item or slice assignment')
+            'object does not support item or slice assignment',
+        )
 
         __setattr__ = _handler(
-            '__guarded_setattr__',
-            'attribute-less object (assign or del)')
+            '__guarded_setattr__', 'attribute-less object (assign or del)'
```

Comment:
That's inconsistent: only because the arguments are on one line, there is no trailing comma?!